packages/@sanity/types/package.json

```diff
@@ -49,16 +49,19 @@
     "watch": "pkg-utils watch"
   },
   "dependencies": {
-    "@sanity/client": "^6.24.1",
-    "@types/react": "^18.3.5"
+    "@sanity/client": "^6.24.1"
   },
   "devDependencies": {
     "@repo/package.config": "workspace:*",
     "@repo/test-config": "workspace:*",
     "@sanity/insert-menu": "1.0.16",
+    "@types/react": "^18.3.5",
     "@vitejs/plugin-react": "^4.3.4",
     "react": "^18.3.1",
     "rimraf": "^5.0.10",
     "vitest": "^2.1.8"
+  },
+  "peerDependencies": {
+    "@types/react": "*"
```

Comment:
not `18 || 19` tho?

Reply:
Good catch! That's more accurate, as those are the versions we test against

Reply:
fixed!